generators/server/templates/build.gradle.ejs

```diff
@@ -596,6 +596,9 @@ dependencies {
     <%_ if (messageBroker === 'kafka') { _%>
     testImplementation "org.testcontainers:kafka"
     <%_ } _%>
+    <%_ if (cacheProvider === 'redis') { %>
```

Comment:
For maven we just use `org.testcontainers:testcontainers`. `database-commons` depends on `testcontainers` so not problem with that, just curious why explicitly using `database-commons` here

Reply:
@atomfrede : I wasn't sure to use `org.testcontainers.database-commons` or `org.testcontainers.testcontainers`. I see that in [Cassandra we explicitly use `database-commons`](https://github.com/jhipster/generator-jhipster/blob/3b008bcbfcbdd190dd951b36743285bbe6a680b6/generators/server/templates/pom.xml.ejs#L485-L495). This is why I asked bsidesup above the difference between the two (do you know? :thinking:). So you mean using `org.testcontainers.testcontainers` is more appropriate. If so do you know why we use `org.testcontainers.database-commons` in Cassandra when using a similar `GenericContainer`? :smile:

Reply:
Cassandra depends on `database-commons`, no need to use it explicitly.

Reply:
@bsideup: You mean [Cassandra module[(https://www.testcontainers.org/modules/databases/cassandra/) depends on `database-commons` right? We don't use that; we use a `GenericContainer` for Cassandra instead and still use `database-common`. My thinking is this is unnecessary probably. :thinking:

Reply:
> For maven we just use `org.testcontainers:testcontainers`. `database-commons` depends on `testcontainers` so not problem with that, just curious why explicitly using `database-commons` here

@atomfrede : So I've replaced it with `org.testcontainers:testcontainers`. Thanks for pointing it out. :smile: